xarray/core/missing.py

```diff
@@ -30,35 +30,30 @@ def __repr__(self):
 
 
 class NumpyInterpolator(BaseInterpolator):
-    '''One-dimensional linear interpolation.
+    """One-dimensional linear interpolation.
 
     See Also
     --------
     numpy.interp
-    '''
-
-    def __init__(self, xi, yi, method='linear', fill_value=None, **kwargs):
+    """
+    def __init__(self, xi, yi, method='linear', fill_value=None, period=None):
```

Comment:
You lost `**kwargs` here.

Reply:
Yes, and I also lost the lines that were checking that kwargs could contain at most the period key :)